### PR TITLE
3 packages from andersfugmann/amqp-client at 2.2.0

### PR DESCRIPTION
**Pull request by andersfugmann**
This pull-request concerns:
-`amqp-client.2.2.0`: Amqp client base library
-`amqp-client-async.2.2.0`: Amqp client library, async version
-`amqp-client-lwt.2.2.0`: Amqp client library, lwt version

---
* Homepage: https://github.com/andersfugmann/amqp-client
* Source repo: git+https://github.com/andersfugmann/amqp-client.git
* Bug tracker: https://github.com/andersfugmann/amqp-client/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/amqp-client-async/amqp-client-async.2.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+dev-repo: "git+https://github.com/andersfugmann/amqp-client.git"
+doc: "https://andersfugmann.github.io/amqp-client/amqp-client-async/Amqp_client_async/"
+license: "BSD3"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/amqp-client-async/amqp-client-async.2.2.0/opam]:
```suggestion
  "dune" {>= "1.1"}
```
Dune should not be tagged as a build dependency anymore (see https://github.com/ocaml/opam-repository/pull/14266). Also the `dune-project` at the root of your source file says you require at least dune 1.1.

File: packages/amqp-client-lwt/amqp-client-lwt.2.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+dev-repo: "git+https://github.com/andersfugmann/amqp-client.git"
+doc: "https://andersfugmann.github.io/amqp-client/amqp-client-lwt/Amqp_client_lwt/"
+license: "BSD3"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/amqp-client-lwt/amqp-client-lwt.2.2.0/opam]:
```suggestion
  "dune" {>= "1.1"}
```

File: packages/amqp-client/amqp-client.2.2.0/opam [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+dev-repo: "git+https://github.com/andersfugmann/amqp-client.git"
+doc: "https://andersfugmann.github.io/amqp-client/"
+license: "BSD3"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/amqp-client/amqp-client.2.2.0/opam]:
```suggestion
  "dune" {>= "1.1"}
```

**Comment by kit-ty-kate** [comment]
Otherwise it's good to go

**Comment by andersfugmann** [comment]
Thanks for the comments. I've updated the opam files.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**